test/599-whitewater.py

```diff
@@ -1,3 +1,9 @@
+# node 2356117215
```

Comment:
This node is in Europe, and so won't be in any North America extracts we use to test with. Suggest http://www.openstreetmap.org/node/3134398100 instead?